### PR TITLE
feat(program): add webhook for program notifications

### PR DESCRIPTION
**Pull request by Birkbjo**
[olhttps://jira.dhis2.org/browse/DHIS2-10855](https://jira.dhis2.org/browse/DHIS2-10855)

### CODE REVIEW

File: src/config/field-rules.js [hunk 1]
```diff
@@ -739,6 +756,23 @@ export default new Map([
...
+            operations: [{
+                field: 'deliveryChannels',
+                type: 'CHANGE_VALUE',
+                setValue: (model, fieldConfig) => {
+                    if(fieldConfig) {
+                        fieldConfig.value = model[fieldConfig.name] = ['HTTP']
+                    }
+                }
+            }],
+        },
```
Comment (Mohammer5) [inline src/config/field-rules.js:775]: Duplicate rule, same as the one added above

**Review by Mohammer5 — CHANGES_REQUESTED** [review]

File: src/config/field-rules.js (hunk 1 above)
Comment (Birkbjo) [reply src/config/field-rules.js:775]: Yup, but they're under different models.

Comment (Mohammer5) [reply src/config/field-rules.js:775]: Ah, oops, didn't notice. Yeah I wouldn't put that into a variable either

**Review by Mohammer5 — APPROVED** [review]

File: src/config/field-rules.js (hunk 1 above)
Comment (Birkbjo) [reply src/config/field-rules.js:775]: Not easy to see from GitHub diffs, since that was truncated 😄

**Pull request merged**